### PR TITLE
Add getActivePlayerIndexSet

### PR DESCRIPTION
**Pull request by ryotogashi**

### CODE REVIEW

**Comment by ryotogashi** [comment]
@axross Can you check when you have time?

File: hooks/usePlayerActions/getActivePlayerIndexSet.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+) {
+  const activePlayerSet = new Set<number>();
+
+  for (const streetAction of gameStreetActions) {
+    if (activePlayerSet.has(streetAction.playerIndex)) break;
+
+    activePlayerSet.add(streetAction.playerIndex);
+  }
+
+  return activePlayerSet;
```
Comment (axross) [inline hooks/usePlayerActions/getActivePlayerIndexSet.ts:14]:
```suggestion
  return new Set(streetAction.map(({ playerIndex }) => playerIndex))
```

Comment (axross) [reply hooks/usePlayerActions/getActivePlayerIndexSet.ts:14]: If anything, we don't need this function?

Comment (ryotogashi) [reply hooks/usePlayerActions/getActivePlayerIndexSet.ts]: @axross Thanks for your comments, but it doesn't work. It returns Set of initial action player indexes. Maybe the name should be getInitialActivePlayerIndexSet.

Comment (ryotogashi) [reply hooks/usePlayerActions/getActivePlayerIndexSet.ts]: @axross You are right.

**Pull request closed without merging**